server/src/main/java/org/apache/druid/server/coordinator/rules/LoadRule.java

```diff
@@ -29,29 +33,63 @@
  */
 public abstract class LoadRule implements Rule
 {
+  protected final Map<String, Integer> tieredReplicants;
```

Comment:
Why do these fields need to be protected? They can remain private and the sub-classes can call the getters when needed.